src/QsCompiler/TextProcessor/SyntaxBuilder.fs

```diff
@@ -337,25 +332,26 @@ let internal buildTupleItem validSingle bundle errCode missingCode fallback cont
 /// Returns the parsed symbol name as Some otherwise. 
 /// IMPORTANT: this routines handles the name *only* and does *not* handle whitespace! 
 /// In order to guarantee correct whitespace management, the name needs to be parsed as a term.
-let internal symbolNameLike errCode = 
-    let identifier = 
-        let id = IdentifierOptions(
-                     isAsciiIdStart = isSymbolStart,
-                     isAsciiIdContinue = isSymbolContinuation,
-                     preCheckStart = isSymbolStart,
-                     preCheckContinue = isSymbolContinuation) |> identifier
-        getPosition .>>. id .>>. getPosition |>> fun ((p1, name), p2) -> name, (p1,p2)
-    let whenValid (name : string, range) = 
+let internal symbolNameLike errCode =
+    let id =
```

Comment:
I think it was named `identifier` to avoid confusion with the built-in F# function `id`.

Reply:
That makes sense, although `identifier` is also the name of an FParsec parser that is used in the definition of `id`... I renamed this variable to `ident`.